help/en/releasenotes/current-draft-note.shtml

```diff
@@ -643,9 +643,10 @@
    <h3>Warrants</h3>
         <a id="Wt" name="Wt"></a>
         <ul>
-            <li>SCWarrants brought back to life with a few fixes. 
-			    One of them being that it is once again possible to create SCWarrants. 
-				Another one that train tracking is now working.</li>
+            <li>SCWarrants brought back to life with a few fixes:  
+			    It is once again possible to create SCWarrants. 
+				Train tracking is now working.
+				Opening the edit dialog for an SCWarrant now whows the correct warrant type.</li>
         </ul>
```

Comment:
Note that this is HTML. The spaces will not stay intact when viewed in a web browser. You need to use `<ul>` and `<li>` to make it a list.

```
        <ul>
            <li>SCWarrants brought back to life with a few fixes:  
			    <ul>
                                <li>It is once again possible to create SCWarrants.</li>
				<li>Train tracking is now working.</li>
				<li>Opening the edit dialog for an SCWarrant now whows the correct warrant type.</li>
			    </ul>
            </li>
        </ul>
```